### PR TITLE
fixes #73781

### LINKED ISSUE #73781: Web: right-click with mouse on tree/list opens native and custom context menu
Steps:
* `yarn web`
* right click into explorer root node
* 🐛 you see native and custom menu

image

Looks like the new tree is not preventing default and bubbling of `contextmenu` event like the old tree did?

### PR DESCRIPTION
**Pull request by sbatten**

### CODE REVIEW

**Review by bpasero — CHANGES_REQUESTED** [review]
Not a big fan of this change. I would have expected a fix somewhere closer in the new tree as this does not reproduce with the old tree.

**Comment by sbatten** [comment]
@bpasero that's only because electron doesn't have a native default for contextmenu, but could you clarify what you mean by "closer"

**Comment by bpasero** [comment]
@sbatten yeah I need to check how we used to prevent the default menu back when we did run in the browser, maybe we had code like this which we removed eventually.

File: src/vs/workbench/browser/layout.ts [hunk 1]
```diff
@@ -206,6 +206,11 @@ export abstract class Layout extends Disposable implements IWorkbenchLayoutServi
 		// Prevent workbench from scrolling #55456
 		this._register(addDisposableListener(this.container, EventType.SCROLL, () => this.container.scrollTop = 0));
 
+		// Prevent native context menus in web #73781
```
Comment (isidorn) [inline src/vs/workbench/browser/layout.ts]: fake comment testing pr extension

Comment (isidorn) [inline src/vs/workbench/browser/layout.ts]: fake comment testing pr extension

Comment (isidorn) [reply src/vs/workbench/browser/layout.ts]: second comment

Comment (isidorn) [reply src/vs/workbench/browser/layout.ts]: test

**Comment by sbatten** [comment]
@bpasero since this change only affects web and is trivial to revert, I'm going to merge this. If you want to move it later, feel free

**Comment by bpasero** [comment]
@sbatten ok. I checked out the old sources of when VSCode was running on Azure and cannot find such code there, so we must have done it differently back then.

**Pull request merged**